pkg/networkservice/chains/client/client.go

```diff
@@ -73,7 +74,7 @@ func WithAuthorizeClient(authorizeClient networkservice.NetworkServiceClient) Op
 // NewClient - returns a (1.) case NSM client.
 //             - ctx    - context for the lifecycle of the *Client* itself.  Cancel when discarding the client.
 //             - cc - grpc.ClientConnInterface for the endpoint to which this client should connect
-func NewClient(ctx context.Context, cc grpc.ClientConnInterface, clientOpts ...Option) networkservice.NetworkServiceClient {
+func NewClient(ctx context.Context, connectTo *url.URL, dialOptions []grpc.DialOption, clientOpts ...Option) networkservice.NetworkServiceClient {
```

Comment:
Probably it would be better to add new option instead of passing here a slice.
```go
func WithDialOptions(dialOptions ...grpc.DialOption) Option {...}
```

Reply:
@glazychev-art
Not sure if you are receiving notifications for this, so calling you :)

Reply:
Done